model/EvhrToA.py

```diff
@@ -866,8 +792,8 @@ def _runOneStrip(stripID, scenes, bandDir, stripDir, orthoDir, demDir,
 
         if panSharpen and scenes[0].isMultispectral():
 
-            EvhrToA._runPanSharpening(toaName, scenes[0].getCatalogId(),
-                                      stripID, bandDir, stripDir, orthoDir,
+            EvhrToA._runPanSharpening(toaName, stripID, scenes, bandDir,
```

Comment:
Line 95 of EvhrToaCelery has a call to this as well. It is still using the old args. Are we moving off of the celery implementation or should we be updating that as well?

```python
        if panSharpen and scenes[0].isMultispectral():

            EvhrToA._runPanSharpening(toaName, scenes[0].getCatalogId(),
                                      stripID, bandDir, stripDir, orthoDir,
                                      demDir, toaDir, outSrsProj4,
                                      panResolution, logger)
```